### DISCUSSION TITLE
windows explorer opens a new window when exiting full screen website videos.

### DISCUSSION THREAD

**Discussion by RavnosRavenous** [valinet/ExplorerPatcher#2713] [Q&A]
i just installed explorerpatcher for the first time on 1/15/2024 and on my windows 10 machine it works fine. for the most part it works great on my win11 rig too but every time i watch something full screen, then leave full screen it opens a new file explorer home window. this can lead to multiple windows open at any given time. please help because im enjoying the program otherwise..

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

Please include any other software you have installed that modifies the UI of windows.
